collector/start_service/start_service.go

```diff
@@ -55,25 +59,51 @@ func (c *startServiceCollector) recordProcesses(ch chan<- prometheus.Metric) err
 		if err != nil {
 			return errors.Wrapf(err, "unable to process SAPControl OSProcess data: %v", *process)
 		}
-		ch <- c.MakeGaugeMetric("processes", state, process.Name, strconv.Itoa(int(process.Pid)), process.Textstatus)
+		ch <- c.MakeGaugeMetric(
```

Comment:
stylish I prefer most the 1 liner than this but ok

Reply:
In general, I break down lines that exceed 120 chars. This metric didn't have as many labels as it has now. ;)